smoketests/tests/android/test_android_feedback.py

```diff
@@ -0,0 +1,102 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+import pytest
+from tests import TestCase
+from pages.dashboard import DashboardPage
+from pages.android_feedback_form import AndroidFeedbackFormPage
+
+
+class TestFeedback(TestCase):
+
+    @pytest.mark.nondestructive
+    def test_on_device_links(self, mozwebqa):
```

Comment:
This test and the one below it seem like multiple separate tests bundled into one. Each time you're calling `go_to_feedback_page` should really be a separate testcase.